test/index.js

```diff
@@ -6,323 +6,404 @@ describe('Penpal', () => {
     Penpal.debug = false; // Set to true when debugging tests.
   });
 
-  it('completes a handshake', (done) => {
-    const connection = Penpal.connectToChild({
-      url: `${CHILD_SERVER}/child.html`
-    });
-
-    connection.promise.then(() => {
-      connection.destroy();
-      done();
+  describe('connectToChild without iframe', () => {
+    it('completes a handshake', (done) => {
+      const connection = Penpal.connectToChild({
+        url: `${CHILD_SERVER}/child.html`
+      });
+  
+      connection.promise.then(() => {
+        connection.destroy();
+        done();
+      });
     });
-  });
-
-  it('creates an iframe and add it to document.body', () => {
-    const connection = Penpal.connectToChild({
-      url: `${CHILD_SERVER}/child.html`
+  
+    it('creates an iframe and add it to document.body', () => {
+      const connection = Penpal.connectToChild({
+        url: `${CHILD_SERVER}/child.html`
+      });
+  
+      expect(connection.iframe).toBeDefined();
+      expect(connection.iframe.src).toBe(`${CHILD_SERVER}/child.html`);
+      expect(connection.iframe.parentNode).toBe(document.body);
     });
-
-    expect(connection.iframe).toBeDefined();
-    expect(connection.iframe.src).toBe(`${CHILD_SERVER}/child.html`);
-    expect(connection.iframe.parentNode).toBe(document.body);
-  });
-
-  it('creates an iframe and add it to a specific element', () => {
-    const container = document.createElement('div');
-    document.body.appendChild(container);
-
-    const connection = Penpal.connectToChild({
-      url: `${CHILD_SERVER}/child.html`,
-      appendTo: container
+  
+    it('creates an iframe and add it to a specific element', () => {
```

Comment:
Both this test and the one before it should say "adds it to" instead of "add it to". That's my fault though. Don't feel like you have to fix it, but if you do, that'd be great.

Reply:
Fixed as well